backend/src/main/java/net/es/oscars/sb/nso/NsoProxy.java

```diff
@@ -219,4 +221,57 @@ public static YangPatchWrapper makeDismantleYangPatch(NsoAdapter.NsoOscarsDisman
         return YangPatchWrapper.builder().patch(deletePatch).build();
     }
 
+
+    // NSO live status fdb query stuff
+    public String getLiveStatusFdbInfo(String device) {
+        if(device == null) {
```

Comment:
style comment: leave a space between if and the condition
ie instead of `if(something) {` do `if (something) {`

Reply:
I'll try, but this will be a hard one since it's muscle memory :( let me reformat